components/List.tsx

```diff
@@ -0,0 +1,14 @@
+interface Props {
+  lists: { content: string }[];
+}
```

Comment:
**praise:** We should have Prettier formatting turned on automatically. Later I will add it as a pre-commit hook.

Reply:
What line isn't correctly formatted?

Reply:
I think it is fine on the second look. Gonna add Prettier today as a hook.

Reply:
Okay

Reply:
This solution is way too complex. There is no need to create separate component. I will open PR to your branch. :)

Reply:
What do i remove?